dist/profile/manifests/jenkinscontroller.pp

```diff
@@ -242,6 +247,47 @@
     $jcasc_java_opts = ''
   }
 
+  if $jcasc_final_config['datadog'] {
+    firewall { '901 accept datadog local metrics collection':
+      proto   => 'udp',
+      dport   => $datadog['metrics_collection_port'],
+      iniface => ['! lo', '! docker0'],
+      action  => 'reject',
+    }
+
+    firewall { '902 accept datadog local trace collection':
+      proto   => 'tcp',
+      dport   => $datadog['traces_collection_port'],
+      iniface => 'docker0',
+      action  => 'accept',
+    }
+
+    if $jcasc_final_config['datadog']['collectBuildLogs'] {
+      firewall { '905 accept datadog local jenkins logs collection':
```

Comment:
```suggestion
      firewall { '903 accept datadog local jenkins logs collection':
```

Reply:
(Restore rule index specified before https://github.com/jenkins-infra/jenkins-infra/commit/77c1ccb70c7d3f533e0c77ea5985c9a7e4506813)

Reply:
I chose to change it to avoid confusion:

- The rules are not created on ci.jenkins.io yet, so no constraint on, the order (besides, iptables priorities are easier to manage than Azure NSG priorities)
- We might have to create a few other rules for the metric and traces, So i've let the indexes available (`903` and `904`)
- The rule was `902` previously (and `903` for the traces). Now traces have the `902` priority. So using `905` avoids any confusion on the order and nested conditions.